R/tensor-R6.R

```diff
@@ -175,6 +175,11 @@
       `torch::Tensor`$dispatch(tensor_tanh_(self$pointer))
     },
 
+    tanh_ = function() {
+      tensor_tanh__(self$pointer)
+      invisible(NULL)
```

Comment:
@Athospd I think it's bets to use `invisible(self)` so users can chain method like `x$tanh_()$mean()`. Some old code are still using `invisible(NULL)`.

Reply:
I just added tests for all methods you created.